README.md

```diff
@@ -50,6 +50,21 @@ Lastly be sure to load the custom Jasmine matchers before your tests run like so
   });
 ```
 
+If you would like the promise matchers to flush $httpBackend, $interval, or $timeout automatically, some options are available:
+
+```javascript
+  beforeEach(function() {
+
+    installPromiseMatchers({
+      httpBackend: true,
+      interval: true,
+      timeout: true
```

Comment:
nit: I think `flushHttpBackend` would be a more descriptive name than `httpBackend`

Reply:
I also think we should leave these on by default for now, so we don't have to release a new major.

Reply:
Good point - should have run this by the discussion before opening the PR.

Reply:
No worries 😄 I changed it before merging